### ISSUE TITLE
`between` as default

### ISSUE THREAD

**Issue by Andarist** [krawaller/callbag-from-delegated-event#2]
Having experience with other delegating libraries (i.e. jQuery) it's unexpected to me that by default you match only targets matching the selector and not also their descendants.

Personally I would get rid of `between` option entirely and just make it a default. That unfortunately would also remove mutating the event object with `.matchedElement` - I think most use cases shouldn't need it anyway and if needed it can easily be obtainer in user land with simple helper.

Ofc feel free to keep existing code/semantics if u believe it's better.

**Comment by krawaller** [comment]
Hmm. Good point on the prior art; IIRC, I likely made it this way because my first use case needed just the matching target, and then later on I needed descendants too. I'd definitely be open to flip the default around.

But, why would flipping the default remove mutating the event object?

**Comment by Andarist** [comment]
> But, why would flipping the default remove mutating the event object?

No idea 🤣 it doesnt have to 😉 I still believe this mutating is somewhat unnecessary and makes the operator impure.

Also - I was proposing removing `between` entirely (making it a default) and not flipping the default. Exact match could still be achieved with
```js
const selector = '.pawn'

pipe(
  fromDelegatedEvent(document.getElementBydId('board'), selector, 'click'),
  filter(event => event.target.matches(selector))
)
```

which could even be packaged as separate operator (`fromExactDelegatedEvent` ? 😄 ) in the spirit of composition.

Still - do as you think is appropriate, I'm just exploring possibilities.

**Comment by krawaller** [comment]
Fixed in `2.0.0`.

**Comment by Andarist** [comment]
cool!
